data/rulei/rulei.txt

```diff
@@ -90,6 +90,23 @@ mission "Rulei Planet"
 			`	It's best you don't remain here for too long.`
 				decline
 
+
+mission "Umbral's Reach"
+	invisible
+	landing
+	to complete
+		never
+	to offer
+		has "First Contact: Rulei: offered"
+		has "visited system: L-118"
```

Comment:
Why does this system need to be visited?

Reply:
This system is the first system you pass through in the dark region area (and is the only way you can get through to it). The point of this mission is the vyu-Ir is able to tell you visited the area, and has a comment on it. Realistically, the player would have gone beyond L-118, but L-118 is still one of the Reach's systems.